src/Modules/OrchardCore.Commerce/Drivers/StripeApiSettingsDisplayDriver.cs

```diff
@@ -22,28 +22,28 @@ public class StripeApiSettingsDisplayDriver : SectionDisplayDriver<ISite, Stripe
     private readonly IDataProtectionProvider _dataProtectionProvider;
     private readonly IShellHost _shellHost;
     private readonly ShellSettings _shellSettings;
-    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IHttpContextAccessor _hca;
```

Comment:
This is the name what the Lombiq Dependency injection VS extension suggests, so I replaced it here too.